docs/providers/aws/guide/serverless.yml.md

```diff
@@ -928,6 +928,35 @@ functions:
           enabled: false
 ```
 
+### Kafka
+
+[Kakfa events](../events/kafka.md):
+
+```yaml
+functions:
+  hello:
+    # ...
+    events:
+      - kafka:
+          # See main kafka documentation for various access configuration settings
+          accessConfigurations:
+            # ...
+          # An array of bootstrap server addresses
+          bootstrapServers:
+            - abc3.xyz.com:9092
+            - abc2.xyz.com:9092
+          # name of Kafka topic to consume from
+          topic: MySelfManagedKafkaTopic
+          # Optional, must be in 1-10000 range
```

Comment:
Let's be consistent in how we specify max and min values

Reply:
Fixed in 814789c99055e66cb77606a1be5656e89271a343